man/systemd.network.xml

```diff
@@ -796,6 +796,33 @@
       </variablelist>
   </refsect1>
 
+    <refsect1>
+    <title>[AddressLabel] Section Options</title>
+
+      <para>An <literal>[AddressLabel]</literal> section accepts the
+      following keys. Specify several <literal>[AddressLabel]</literal>
+      sections to configure several addresse labels. IPv6 address labels are
+      used for address selection. See <ulink url="https://tools.ietf.org/html/rfc3484">RFC 3484</ulink>.
+      Precedence is managed by userspace, and only the label itself is stored in the kernel</para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>Label=</varname></term>
+          <listitem>
+            <para> The label for the prefix (an unsigned integer) in the range 0 to 65,535 or 0 to 4,294,967,295.
+            depending on 2 or 4 bytes. 0xffffffff is reserved. This key is mandatory.</para>
```

Comment:
what does that mean: "depending on 2 or 4 bytes"? Can't make sense of it.

If 0xffffffff is reserved, what does that mean? If it is not an acceptable value, then you need to filter it out, and hence document that the valid range for this is 0..4294967294 instead of 0..4294967295